### ISSUE TITLE
[PHP] Error: Undefined index: changelogURL

### ISSUE THREAD

**Issue by Githopp192** [nextcloud/server#19653]
### Steps to reproduce
appears regulary into the Nextcloud log

### Expected behaviour
no php error

### Actual behaviour
php core error

## Server configuration detail

**Operating system:** Linux 3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020 x86_64

**Webserver:** Apache (apache2handler)

**Database:** mysql 5.5.64

**PHP version:** 

7.2.24
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, apache2handler, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, intl, json, ldap, exif, mysqlnd, PDO, Phar, posix, shmop, SimpleXML, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlwriter, xsl, zip, mysqli, pdo_mysql, pdo_sqlite, wddx, xmlreader, apcu, igbinary, imagick, redis, Zend OPcache

**Nextcloud version:** 17.0.3 - 17.0.3.1

**Updated from an older Nextcloud/ownCloud or fresh install:** upgrade from NC 16.0.8

**Where did you install Nextcloud from:** Origin Source

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.3.0
 - activity: 2.10.1
 - analytics: 2.0.0
 - announcementcenter: 3.6.1
 - apporder: 0.9.0
 - audioplayer: 2.9.0
 - bookmarks: 2.3.4
 - bruteforcesettings: 1.5.0
 - calendar: 2.0.1
 - circles: 0.17.10
 - cloud_federation_api: 1.0.0
 - comments: 1.7.0
 - contacts: 3.1.8
 - cookbook: 0.5.7
 - cospend: 0.3.3
 - dashboardcharts: 0.1.3
 - data_request: 1.4.0
 - dav: 1.13.0
 - deck: 0.8.0
 - dicomviewer: 1.2.1
 - drawio: 0.9.5
 - event_update_notification: 1.0.1
 - extract: 1.2.3
 - federatedfilesharing: 1.7.0
 - federation: 1.7.0
 - files: 1.12.0
 - files_antivirus: 2.2.1
 - files_downloadactivity: 1.6.0
 - files_mindmap: 0.0.17
 - files_pdfviewer: 1.6.0
 - files_rightclick: 0.15.1
 - files_sharing: 1.9.0
 - files_trashbin: 1.7.0
 - files_versions: 1.10.0
 - files_videoplayer: 1.6.0
 - firstrunwizard: 2.6.0
 - flowupload: 0.1.8
 - gallery: 18.4.0
 - groupfolders: 5.0.5
 - issuetemplate: 0.6.0
 - logreader: 2.2.0
 - lookup_server_connector: 1.5.0
 - maps: 0.1.5
 - nextcloud_announcements: 1.6.0
 - notes: 3.1.5
 - notifications: 2.5.0
 - oauth2: 1.5.0
 - onlyoffice: 4.1.4
 - ownbackup: 19.9.0
 - passman: 2.3.5
 - password_policy: 1.7.0
 - polls: 1.3.0
 - privacy: 1.1.0
 - provisioning_api: 1.7.0
 - quicknotes: 0.2.3
 - quota_warning: 1.6.0
 - radio: 0.6.6
 - rainloop: 6.1.0
 - ransomware_protection: 1.5.1
 - recommendations: 0.5.0
 - serverinfo: 1.7.0
 - sharebymail: 1.7.0
 - sharerenamer: 2.7.3
 - socialsharing_diaspora: 1.0.5
 - socialsharing_email: 1.0.6
 - socialsharing_facebook: 1.0.5
 - socialsharing_twitter: 1.0.5
 - spreed: 7.0.2
 - support: 1.0.1
 - suspicious_login: 3.1.0
 - systemtags: 1.7.0
 - tasks: 0.11.3
 - terms_of_service: 1.3.1
 - text: 1.1.1
 - theming: 1.8.0
 - twofactor_backupcodes: 1.6.0
 - twofactor_totp: 4.1.2
 - updatenotification: 1.7.0
 - viewer: 1.2.0
 - workflowengine: 1.7.0
Disabled:
 - admin_audit
 - breezedark
 - carnet
 - dashboard
 - drop_account
 - encryption
 - external
 - files_accesscontrol
 - files_ebookreader
 - files_external
 - files_fulltextsearch
 - files_markdown
 - fulltextsearch
 - fulltextsearch_elasticsearch
 - impersonate
 - joplin
 - jsloader
 - passwords
 - ransomware_detection
 - richdocuments
 - socialsharing_googleplus
 - survey_client
 - user_ldap
 - weather

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "memcache.local": "\\OC\\Memcache\\APCu",
    "filelocking.enabled": true,
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 0,
        "dbindex": 0,
        "timeout": 1.5
    },
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
       
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "overwrite.cli.url": "https:\/\/www.xxx.xx",
    "htaccess.RewriteBase": "\/",
    "overwriteprotocol": "https",
    "dbtype": "mysql",
    "version": "17.0.3.1",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "maintenance": false,
    "theme": "",
    "loglevel": 1,
    "updater.release.channel": "stable",
    "auth.bruteforce.protection.enabled": true,
    "check_for_working_htaccess": true,
    "data-fingerprint": "xxxx",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpmode": "smtp",
    "mail_smtpauthtype": "LOGIN",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpsecure": "tls",
    "mail_smtpauth": 1,
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "xx",
    "session_lifetime": 1200,
    "session_keepalive": false,
    "logtimezone": "xx\/xxx",
    "logfile": "\/media\/log\/nextcloud.log",
    "knowledgebaseenabled": false,
    "log_rotate_size": xxx,
    "onlyoffice": {
        "verify_peer_off": true
    },
    "mail_sendmailmode": "smtp"
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** false

**Are you using an external user-backend, if yes which one:** no

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.116 Safari/537.36 Edg/80.0.361.57

**Operating system:** Windows 10

## Logs

<details><summary>Web server error log</summary>

```

```
</details>

<details><summary>Nextcloud log</summary>

```
[PHP] Error: Undefined index: changelogURL at /var/www/html/nextcloud/core/Controller/WhatsNewController.php#91

GET /ocs/v2.php/core/whatsnew?format=json
from 88.151.146.103 by admin at 2020-02-26T09:22:59+01:00
```
</details>

<details><summary>Browser log</summary>

</details>

**Comment by kesselb** [comment]
Thanks for reporting 👍

Please use the search before creating a new report 🔎

**Comment by nooblag** [comment]
I also get this error in Nextcloud 18.0.1. If this has been closed, was it fixed?

**Comment by kesselb** [comment]
I closed that issue because it's a duplicate of: https://github.com/nextcloud/server/issues/19490 

A fix will come to 18.0.2, 17.0.4, 16.0.10.

**Comment by nooblag** [comment]
Okay, thanks. ;)
